README.md

```diff
@@ -157,7 +156,7 @@ the cloud, sort-of).  So what we're doing in this step is creating the build pip
   * CONTAINER_MEMORY is the amount of memory (in MiB) to reserve for this application (e.g. 512).
   * LISTENER_RULE_PRIORITY is the priority of the the rule that gets created in the ALB.  While these won't ever conflict, ALB requires a unique number across all apps that share the ALB.  See [Application specifics](#application-specifics)
   * (optional) REPO_BRANCH is the branch name for the repo - MUST NOT CONTAIN SLASHES!
-  * (optional) PREFIX is what goes in front of the URI of the application.  Defaults to OWNER but for the "real" riglet should be set to blank (e.g. `PREFIX=`)
+  * (optional) SUBDOMAIN is placed in front of the DOMAIN configured in the .make file when generating ALB listener rules.  Defaults to REPO.
```

Comment:
do you want to include the params down in the table where we spit out listener order, repo etc?